### PR TITLE
fixed video head position on mobile

### PR DESCRIPTION
**Pull request by aleksspeaker**

### CODE REVIEW

File: dev/conversation/styles.scss [hunk 1]
```diff
@@ -267,9 +266,13 @@
     .vjs-video-wrap,
     .video-js {
       max-width: 100%;
-      height: 400px;
+      height: auto;
```
Comment (dmitrij-borchuk) [inline dev/conversation/styles.scss]: Do we really need it?

File: dev/conversation/styles.scss [hunk 2]
```diff
@@ -267,9 +266,13 @@
...
     }
 
+    .video-js .vjs-tech {
+      position: auto;
```
Comment (dmitrij-borchuk) [inline dev/conversation/styles.scss]: "position" doesn't have value "auto"

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/conversation/styles.scss (hunk 1 above)
Comment (aleksspeaker) [reply dev/conversation/styles.scss]: Yes.

File: dev/conversation/styles.scss (hunk 2 above)
Comment (aleksspeaker) [reply dev/conversation/styles.scss]: Changed to static(default)

File: dev/conversation/styles.scss [hunk 3]
```diff
@@ -267,7 +266,11 @@
     .vjs-video-wrap,
     .video-js {
       max-width: 100%;
-      height: 400px;
+      height: auto;
+    }
+
+    .video-js .vjs-tech {
+      position: static;
```
Comment (dmitrij-borchuk) [inline dev/conversation/styles.scss]: Remove it if it is default

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/conversation/styles.scss (hunk 3 above)
Comment (aleksspeaker) [reply dev/conversation/styles.scss]: I can't it doesn't work without it.

Comment (aleksspeaker) [reply dev/conversation/styles.scss]: Looks like this setting comes from outside.

Comment (aleksspeaker) [reply dev/conversation/styles.scss]: It sets it to absolute by default.

**Comment by dmitrij-borchuk** [comment]
Please remove comments

**Review by dmitrij-borchuk — APPROVED** [review]

**Pull request merged**